arrows/tests/test_resection_camera.h

```diff
@@ -47,6 +46,64 @@ test_resection_camera(
                          estimated_camera->center(), center_tolerance );
 }
 
+// ----------------------------------------------------------------------------
+// Helper to test camera resection
+template < typename... Args >
+void
+test_resection_camera(
+  landmark_map_sptr const& landmarks, feature_track_set_sptr const& tracks,
+  frame_id_t test_frame, camera_perspective_sptr const& expected_camera,
+  double center_tolerance, double rotation_tolerance,
+  Args... args )
+{
+  resection_camera algo;
+  test_algo_resection_camera( algo,
+    landmarks, tracks, test_frame, expected_camera,
+    center_tolerance, rotation_tolerance,
+    args... );
+}
+
+// ----------------------------------------------------------------------------
+// Helper to test camera resection
+template < typename... Args >
+void
+test_resection_camera_cfg(
+  landmark_map_sptr const& landmarks, feature_track_set_sptr const& tracks,
+  frame_id_t test_frame, camera_perspective_sptr const& expected_camera,
+  double center_tolerance, double rotation_tolerance,
+  Args... args )
+{
+  resection_camera algo;
+  auto cfg = algo.get_configuration();
```

Comment:
"config" is six times more prevalent; it would be nice to continue that trend...

Reply:
using config now